src/main/java/seedu/ccacommander/ui/EventCard.java

```diff
@@ -18,7 +18,7 @@ public class EventCard extends UiPart<Region> {
      * As a consequence, UI elements' variable names cannot be set to such keywords
      * or an exception will be thrown by JavaFX during runtime.
      *
-     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on CcaCommander level 4</a>
```

Comment:
Should this stay as AddressBook since it's referring to the AB4 Repo?
```suggestion
     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
```

Reply:
resolved